core/src/main/scala/cats/data/Streaming.scala

```diff
@@ -524,7 +524,7 @@ sealed abstract class Streaming[A] extends Product with Serializable { lhs =>
    *
    * For example:
    *
-   *   Streaming(1, 2, 3, 4, 5, 6, 7).takeWhile(n => n != 4)
+   *   Streaming(1, 2, 3, 4, 5, 6, 7).dropWhile(n => n != 4)
```

Comment:
Now that we have sbt-doctest in place, we should change these to be compiler-verified!